src/ui/components/pages/otherAccounts.tsx

```diff
@@ -51,6 +53,11 @@ export function OtherAccountsPage() {
     ])
   );
 
+  const addAccount = useCallback(() => {
```

Comment:
Usually there needs to be a real reason for using `useCallback` or `useMemo` instead of using it for everywhere upfront without measuring if it makes any difference :-)